src/components/MovieDetail/MovieDetail.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './MovieDetail.css'
+
+export default function MovieDetail({
+    movieImageUrl,
+    movieName,
+    genres,
+    releaseDate,
+    score,
+    duration,
+    description }) {
+
+    /*     const handleGenreClick = (movie) => {
+            onclickEvent(movie)
+        }; */
+
+    return (
+        <>
+            <div className="md-container">
+                <div className="md-container-image">
+                    <img src={movieImageUrl} alt="movie-poster" className="md-image" />
+                </div>
+                <div className="md-container-info">
+                    <div className="md-row md-icon">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="29" height="30" viewBox="0 0 29 30" fill="none">
+                            <circle cx="18.5" cy="10.5" r="9.5" stroke="#F65261" strokeWidth="2" />
+                            <path d="M10.5 19.5L1.5 28.5" stroke="#F65261" strokeWidth="2" strokeLinecap="square" />
+                        </svg>
+                    </div>
+                    <div className="md-row">
+                        <h1 className="md-movie-name">{movieName}</h1>
+                        <span className="md-score">{score}</span>
+                    </div>
+                    <div className="md-row">
+                        <h3 className="md-genres">{genres}</h3>
```

Comment:
Update genres, currently they are together:
<img width="552" alt="image" src="https://github.com/edwarfuentes97/epam-react-movies/assets/16781872/a433e0e6-a04c-41e0-9524-b56ccfd00266">

quickest solution `{genres.join(' ')}`

Reply:
not the cleanest, tho. Might be better to iterate each genre as a `span` element and add some spacing with CSS.